### DISCUSSION TITLE
A score not being submitted?

### DISCUSSION THREAD

**Discussion by chayleaf** [ppy/osu#24487] [Q&A] [answered]
I usually play with HDRD, and it submits on Lazer-specific leaderboards just fine (ex: my top score on [this map](https://osu.ppy.sh/beatmapsets/777877#taiko/1633996))

But when I tried submitting a score on [this map](https://osu.ppy.sh/beatmapsets/663341#taiko/1404094) (play ID or whatever is returned by `/extra-pages/historical` endpoint: 459937447), I tried again (play ID: 459937944), it didn't work, instead, only an older play with only difficulty adjust mod enabled and over 2x less score remained. It doesn't seem like it works on other maps either, though my old scores still remain.

Is this a regression, a maintenance thing, or an intended change? I think this changed this or last week.

**Comment by orangixx** [comment]
I _believe_ leaderboards are currently locked, because my personal best won't update, even if I haven't played the map before. They changed osu!std scoring algorithm to scorev2 and probably will reset soon. Your score _is_ submitting and is showing up on your profile-
![](https://github.com/ppy/osu/assets/41548573/5c96ff5c-d226-433e-9ac8-efc6561a5f25)
so the problem is online.

Try setting a score on a map you haven't played before. It still won't update.

**Reply by bdach** [reply]
This is completely bogus and unfounded. Please don't answer questions you don't know answers to. There is no "leaderboard lock" in place. At least not an intentional one.

**Comment by bdach** [comment, marked as answer]
We've identified an issue with score processing, The relevant server component was restarted, new scores should be hopefully processing fine. As for past scores that are stuck, please give us 24-48 hours to catch up.

**Reply by bdach** [reply]
Update: Background reprocessing is currently being run on scores from the past 3 months or so, and it should conclude within the next 12 hours.

**Reply by bdach** [reply]
The score reprocessing [should have terminated](https://github.com/ppy/osu/issues/24502#issuecomment-1678345601) by now. If something still hasn't processed by now please let us know.
